src/main/java/views/add_edit_epense_views/EditExpenseV.java

```diff
@@ -102,27 +104,36 @@ public void actionPerformed(ActionEvent evt) {
                 if (recurExpense.getName().equals(selectedExpense)) {
                     // Set isRecurringCheckBox checked if the expense being edited was a recurring expense
                     isRecurringCheckBox.setSelected(true);}}
-        }
+        } else if (selectedExpense == null) {
+            JOptionPane.showMessageDialog(this, "Please select an expense you wish to edit.");}
```

Comment:
I think this should go inside the <if (evt.getSource() == categoryCombo)> clause.

Reply:
resolve as not relevant